### PR TITLE
Added pull request template with code and content change checklist

### PR DESCRIPTION
**Pull request by keithkennedyHO**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+# Content change 
+I can confirm:
+- [ ] Content is suitable to open source
+- [ ] Content meets the content standards
+- [ ] Content does not relate to unreleased gov policy
+- [ ] Content is appropriately tagged
+- [ ] Content does not refer to anti-fraud mechanisms
+- [ ] Content does not include any code or configuration changes (excluding frontmatter information)
+- [ ] Content does not include sensitive business logic
```
Comment (edhamiltonHO) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: Can scratch 'content is appropriately tagged' - it is part of the content standards

```suggestion
- [ ] Content does not include any code or configuration changes (excluding frontmatter information)
- [ ] Content meets the content standards
e.g. [Writing a principle](https://ho-cto.github.io/engineering-guidance-and-standards/docs/standards/writing-a-principle/) and [Writing a standard](https://ho-cto.github.io/engineering-guidance-and-standards/docs/standards/writing-a-standard/)
- [ ] Content is suitable to open source
i.e.
- Content does not relate to unreleased gov policy
- Content does not refer to anti-fraud mechanisms
- Content does not include sensitive business logic
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Is this pull request a content or a code change?
+
+# Code change
+I can confirm:
+- [ ] Manual accessibility testing has been performed (if appropriate)
```
Comment (edhamiltonHO) [inline .github/PULL_REQUEST_TEMPLATE.md:5]:
```suggestion
- [ ] This change will not change layouts, page structures or anything else that might impact accessibility
or
- [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
(If the change might impact accessibility then please add some further information here)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Is this pull request a content or a code change?
+
+# Code change
+I can confirm:
+- [ ] This change will not change layouts, page structures or anything else that might impact accessibility
+or
+- [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
+(If the change might impact accessibility then please add some further information here)
```
Comment (edhamiltonHO) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: Sorry, just rethought this
```suggestion
## Accessibility considerations
- [ ] This change will not change layouts, page structures or anything else that might impact accessibility
or
- [ ] This change might impact accessibility, automated aXe tests cover the impact
or
- [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
(If the change might impact accessibility then please add some further information here)
```

**Comment by edhamiltonHO** [comment]
Looks like a basis, will be happy to approve once resolve the breaking tests

**Review by edhamiltonHO — APPROVED** [review]

**Review by edhamiltonHO — APPROVED** [review]
Looks good to me

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+# Content change 
+I can confirm:
+- [ ] Content does not include any code or configuration changes (excluding frontmatter information)
+- [ ] Content meets the content standards
+e.g. [Writing a principle](https://ho-cto.github.io/engineering-guidance-and-standards/docs/standards/writing-a-principle/) and [Writing a standard](https://ho-cto.github.io/engineering-guidance-and-standards/docs/standards/writing-a-standard/)
+- [ ] Content is suitable to open source
+i.e.
+- Content does not relate to unreleased gov policy
+- Content does not refer to anti-fraud mechanisms
+- Content does not include sensitive business logic
```
Comment (jeff-horton-ho-sas) [inline .github/PULL_REQUEST_TEMPLATE.md:22]:
```suggestion
- [ ] Content is suitable to open source, i.e.:
    - Content does not relate to unreleased gov policy
    - Content does not refer to anti-fraud mechanisms
    - Content does not include sensitive business logic
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+# Code change
+I can confirm:
+## Accessibility considerations
+- [ ] This change will not change layouts, page structures or anything else that might impact accessibility
+or
+- [ ] This change might impact accessibility, automated aXe tests cover the impact
+or
+- [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
+(If the change might impact accessibility then please add some further information here)
```
Comment (jeff-horton-ho-sas) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: When rendered in GH the `or` is appended to the previous line, so isn't obvious. Add line breaks to have them render as a new paragraph.
```suggestion
- [ ] This change will not change layouts, page structures or anything else that might impact accessibility

or
- [ ] This change might impact accessibility, automated aXe tests cover the impact

or
- [ ] This change might impact accessibility and is not covered by automated aXe tests - manual testing has been performed
(If the change might impact accessibility then please add some further information here)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+Is this pull request a content or a code change?
```
Comment (jeff-horton-ho-sas) [inline .github/PULL_REQUEST_TEMPLATE.md:1]:
```suggestion
Is this pull request a content or a code change? (Please fill in the relevant section and delete the other)
```

**Review by jeff-horton-ho-sas — APPROVED** [review]

**Pull request merged**